src/test/java/dev/fastgql/FastGQLServerMySQLTest.java

```diff
@@ -35,31 +45,25 @@
 public class FastGQLServerMySQLTest {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(FastGQLServerMySQLTest.class);
-  private String deploymentID;
-
   private static final int port = 8081;
-
   private static Network network = Network.newNetwork();
-
   private static KafkaContainer kafkaContainer = new KafkaContainer().withNetwork(network);
-
   private static MySQLContainer<?> mysqlContainer =
       new MySQLContainer<>("fastgql/mysql-testcontainers:latest")
           .withNetwork(network)
           .withNetworkAliases("mysql")
           .withUsername("debezium")
           .withPassword("dbz");
-
   private static DebeziumContainer debeziumContainer =
       new DebeziumContainer("1.0")
           .withNetwork(network)
           .withKafka(kafkaContainer)
           .withLogConsumer(new Slf4jLogConsumer(LOGGER))
           .dependsOn(kafkaContainer);
+  private final int customersStartOffset = 5;
```

Comment:
this is very specific, maybe we can just subscribe to latest message, not from the start of the topic